### PR TITLE
Answers.md uploaded with questions answers and classes.js completed

### PR DESCRIPTION
**Pull request by dragonbegin**
The Answers.md uploaded with questions answers and classes.js completed.

### CODE REVIEW

File: Answers.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.map won't change the original array but create a new one.
+
+2. What is the difference between a function and a method?
+
+Functions are block of code that can be call whenever desire.
+Method is also a function but it has been hosted inside and Object and given a key name mostly.
+
+3. What is closure?
+
+A Closure is the combined operation of a function with his environment.
```
Comment (ari-mercado) [inline Answers.md:19]: This is correct, but more technically, a closure is a nested function that has access to its outer function's variables.  Closures store references to the outer function's variables even after function invocation has completed.

File: challenges/objects-arrays.js [hunk 2]
```diff
@@ -8,27 +8,46 @@
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
 
+let tyrannosaurus = {
+  name: 'tyrannosaurus',
+  diet: 'carnivorous',
+  weigh: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceous',
+  'roar': function () {
```
Comment (ari-mercado) [inline challenges/objects-arrays.js:17]: Quotes on *roar* are optional. But in general it is good practice to make all the keys the same within each object

File: challenges/objects-arrays.js [hunk 3]
```diff
@@ -97,29 +133,34 @@ The zoos need a list of all their animal's names (names only, not scientific) co
...
+  return total += animal.population;
+},0);
 console.log(populationTotal);
 
-
 /* 
 
 Stretch: If you haven't already, convert your array method callbacks into arrow functions.
 
 */
```
Comment (ari-mercado) [inline challenges/objects-arrays.js:165]: Nice job!

**Review by ari7946 — APPROVED** [review]
Great job, Juan!

**Pull request merged**